client/me/purchases/membership-item/index.tsx

```diff
@@ -40,6 +40,8 @@ const MembershipTerms = ( { subscription }: { subscription: MembershipSubscripti
 
 const SiteLink = ( { subscription }: { subscription: MembershipSubscription } ) => {
 	const translate = useTranslate();
+	const lastSlashInUrl = subscription.site_url.lastIndexOf( '/' );
```

Comment:
This feels yucky to me but I don't see another way other than updating the end-point response. Any suggestions?

Reply:
If you wanted to, you could combine the lines:

`const siteUrl = subscription.site_url.replace( /^(https?:|)\/\//,'' );`

Reply:
nice! yeah, I'll clean that up.